Dockerfile

```diff
@@ -1,8 +1,8 @@
 # syntax=docker/dockerfile:experimental
 
 # ============= Setting up base Stage ================
-# Set required AVALANCHE_VERSION parameter in build image script
-ARG AVALANCHE_VERSION
+# Set required AVALANCHEGO_VERSION parameter in build image script
```

Comment:
I think we should go the other way around here, let's keep the Dockerfile name unchanged since it may be referenced externally.

Reply:
yup good idea